pkgs/applications/networking/browsers/misc/widevine-cdm.nix

```diff
@@ -1,34 +1,74 @@
 { lib
 , stdenv
 , fetchzip
+, fetchurl
+, fetchFromGitHub
+, squashfsTools
+, curl
+, python3
 }:
 
 stdenv.mkDerivation rec {
   pname = "widevine-cdm";
   version = "4.10.2710.0";
+  lacrosVersion = "120.0.6098.0";
 
-  src = fetchzip {
+  widevine-installer = if stdenv.isAarch64 then stdenv.mkDerivation rec {
+    name = "widevine-installer";
+    src = fetchFromGitHub {
+      owner = "AsahiLinux";
+      repo = "${name}";
+      rev = "7a3928fe1342fb07d96f61c2b094e3287588958b";
+      sha256 = "sha256-XI1y4pVNpXS+jqFs0KyVMrxcULOJ5rADsgvwfLF6e0Y=";
+    };
+
+    installPhase = ''
+      mkdir -p "$out/bin/"
+      cp widevine_fixup.py "$out/bin/"
+    '';
+  } else null;
+
+  src = if stdenv.isAarch64 then fetchurl {
+    url = "https://commondatastorage.googleapis.com/chromeos-localmirror/distfiles/chromeos-lacros-arm64-squash-zstd-${lacrosVersion}";
+    hash = "sha256-OKV8w5da9oZ1oSGbADVPCIkP9Y0MVLaQ3PXS3ZBLFXY=";
+  } else fetchzip {
     url = "https://dl.google.com/widevine-cdm/${version}-linux-x64.zip";
     hash = "sha256-lGTrSzUk5FluH1o4E/9atLIabEpco3C3gZw+y6H6LJo=";
     stripRoot = false;
   };
 
+  buildInputs = [ squashfsTools curl python3];
+
+  unpackPhase = if stdenv.isAarch64 then ''
+    curl -# -o lacros.squashfs "file://$src"
+    unsquashfs -q lacros.squashfs 'WidevineCdm/*'
+    python3 ${widevine-installer}/bin/widevine_fixup.py squashfs-root/WidevineCdm/_platform_specific/cros_arm64/libwidevinecdm.so libwidevinecdm.so
```

Comment:
This could be `$widevineInstaller` once `widevine-installer` is `widevineInstaller`.